feat: document Deep Agents Code plugins (#4905)
langchain-ai/docs: src/oss/deepagents/code/quickstart.mdx

        - `/copy`: Copy the latest assistant message to the clipboard.
        - `/threads`: Browse and resume previous conversation threads.
        - `/mcp [login <server> | reconnect]`: Show active MCP servers and tools. `login <server>` runs the OAuth flow for a server; `reconnect` loads deferred logins.

>         - `/plugins`: Manage [plugins and marketplaces](/oss/deepagents/code/plugins). Experimental; requires `DEEPAGENTS_CODE_EXPERIMENTAL=1`.

Reviewer: ```suggestion
        - `/plugins`: Manage [plugins and marketplaces](/oss/deepagents/code/plugins).
```

Reviewer: Putting in this suggestion assuming that we drop the experimental gate

        - `/notifications`: Configure startup warning preferences.

>         - `/reload`: Re-read `.env` files, refresh configuration, and re-discover skills without restarting. When experimental plugins are enabled, also reloads plugin skills and plugin MCP servers. Conversation state is preserved. See [`DEEPAGENTS_CODE_` prefix](/oss/deepagents/code/configuration#deepagents_code_-prefix) for override behavior.

Reviewer: ```suggestion
        - `/reload`: Re-read `.env` files, refresh configuration, and re-discover skills without restarting. This also reloads plugin components like skills or MCP config. Conversation state is preserved. See [`DEEPAGENTS_CODE_` prefix](/oss/deepagents/code/configuration#deepagents_code_-prefix) for override behavior.
```

Reviewer: Another eager suggestion
